index.js

```diff
@@ -41,7 +41,11 @@ module.exports = (urlString, opts) => {
 
 	// Remove duplicate slashes
 	if (urlObj.pathname) {
-		urlObj.pathname = urlObj.pathname.replace(/\/{2,}/g, '/');
+		if (/(http:)?(https:)/.test(urlObj.pathname)) {
```

Comment:
A simplification of your regex could be /https?/ 🙂

Reply:
Ha, you are right. I don't know why i completely missed it!